### PR TITLE
[TwigBundle] Fix deprecated use of FlattenException

### PR DESCRIPTION
**Pull request by alOneh**
| Q | A |
| --- | --- |
| Bug fix? | no |
| New feature? | no |
| BC breaks? | no |
| Deprecations? | no |
| Tests pass? | yes |
| Fixed tickets | - |
| License | MIT |
| Doc PR | - |

### CODE REVIEW

File: src/Symfony/Bundle/TwigBundle/Controller/ExceptionController.php [hunk 1]
```diff
@@ -12,7 +12,7 @@
 namespace Symfony\Bundle\TwigBundle\Controller;
 
 use Symfony\Bundle\FrameworkBundle\Templating\TemplateReference;
-use Symfony\Component\HttpKernel\Exception\FlattenException;
+use Symfony\Component\Debug\Exception\FlattenException;
```
Comment (nicolas-grekas) [inline src/Symfony/Bundle/TwigBundle/Controller/ExceptionController.php]: BC break

File: src/Symfony/Bundle/TwigBundle/Controller/PreviewErrorController.php [hunk 2]
```diff
@@ -11,7 +11,7 @@
 
 namespace Symfony\Bundle\TwigBundle\Controller;
 
-use Symfony\Component\HttpKernel\Exception\FlattenException;
+use Symfony\Component\Debug\Exception\FlattenException;
```
Comment (nicolas-grekas) [inline src/Symfony/Bundle/TwigBundle/Controller/PreviewErrorController.php]: :+1:

File: src/Symfony/Bundle/TwigBundle/Tests/Controller/ExceptionControllerTest.php [hunk 3]
```diff
@@ -13,7 +13,7 @@
 
 use Symfony\Bundle\TwigBundle\Tests\TestCase;
 use Symfony\Bundle\TwigBundle\Controller\ExceptionController;
-use Symfony\Component\HttpKernel\Exception\FlattenException;
+use Symfony\Component\Debug\Exception\FlattenException;
```
Comment (nicolas-grekas) [inline src/Symfony/Bundle/TwigBundle/Tests/Controller/ExceptionControllerTest.php]: :+1:

**Comment by alOneh** [comment]
Reverted the BC break

**Comment by nicolas-grekas** [comment]
:+1:

**Comment by stof** [comment]
:+1:

**Comment by nicolas-grekas** [comment]
Thank you @alOneh.

**Comment by xabbuh** [comment]
@philgagnon12 That was not possible to not break backward compatibility (otherwise old exceptions not extending the one from the Debug component would not have been able to be dealt with).

**Pull request merged**